subgraph/pipelines/alpha-calibnet.yaml

```diff
@@ -1,5 +1,4 @@
 name: alpha-calibnet
-version: 6
```

Comment:
This change is not strictly needed, but I think it's better to remove the `version` field as it's out of date with the latest pipeline version on Goldsky and the CLI does not require this field to be present.